docker/docker-compose.yml

```diff
@@ -1,9 +1,11 @@
-version: '3.1'
+version: '2.4' # v3 doesn't support depends_on
```

Comment:
Think it does support it but not in `condition:` form: https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on

Reply:
Yep; you're right. They removed the feature because in Docker Swarm each app is supposed to just retry until it works, instead of quitting if there's a problem, but that's not how HOCS is architected.